builder/ConfTargets.cmake

```diff
@@ -68,6 +68,8 @@ else()
   set ( MFX_1_34_OPTIONS_ALLOWED OFF )
 endif()
 
+option( MFX_ENABLE_AENC "Enabled AENC extension?" ON)
```

Comment:
No! You can _not_ enable enctools by default till you rely on closed source binary (libaenc.a).